proxy/src/lib.rs

```diff
@@ -173,6 +182,10 @@ where
             inbound_listener.local_addr(),
             config.private_forward
         );
+        info!(
+            "serving scrapable metrics on {:?}",
```

Comment:
I recall a debate about "scrapable" vs "Prometheus".  did we settle on "scrapable"?

Reply:
I meant to change this but must've forgotten --- do note though that this string is by analogy to the message the controller prints.

Reply:
The word "scrapable" still shows up a few times in comments in this PR.  Should we update those too?